### DISCUSSION TITLE
Any plans to use pyproject.toml instead of pixi.toml?

### DISCUSSION THREAD

**Discussion by codegod100** [prefix-dev/pixi#618] [Q&A]
I was under the impression pyproject.toml was the "standard" config file

**Comment by baszalmstra** [comment]
For python projects that's definitely the case but for other ecosystems (C++, Rust, etc) its not. We have been discussing allowing you to define the data in a `pixi.toml` in a `pyproject.toml` as well for Python projects. See #79
